TelosLabs/giving-connection
# Feature| Bio Change

### Context
Changing the Bio section 
### What changed
I made some changes to existing bios and added in Mollys and Asthas profiles. 
### How to test it
On the About Us page on the website, scroll down to see two new profiles added with the pictures. 
### References
![image](https://github.com/TelosLabs/giving-connection/assets/119626526/1be5db4a-b881-45b8-86f9-4bdf0a605427)
![image](https://github.com/TelosLabs/giving-connection/assets/119626526/8d7ea7ef-bca8-499e-b80c-acba6db6a881)
![image](https://github.com/TelosLabs/giving-connection/assets/119626526/0208d9fb-a4fa-49bc-94c1-934654b51ef3)
![image](https://github.com/TelosLabs/giving-connection/assets/119626526/d001b05b-2695-4666-a55c-c27a6ba0b584)
![image](https://github.com/TelosLabs/giving-connection/assets/119626526/635598f8-26cf-48bd-aab6-bb7986e56fae)

[[ClickUp ticket](url)
](https://app.clickup.com/t/85yx52u86)
